includes/Core/Util/Activation_Notice.php

```diff
@@ -96,8 +96,7 @@ function( $hook_suffix ) {
 				 */
 				unset( $_GET['activate'] ); // phpcs:ignore WordPress.Security.NonceVerification, WordPress.VIP.SuperGlobalInputUsage
 
-				$this->assets->enqueue_fonts();
-				$this->assets->enqueue_asset( 'googlesitekit-admin-css' );
+				Assets::enqueue_styles( 'googlesitekit-admin-css' );
 				$this->assets->enqueue_asset( 'googlesitekit-activation' );
```

Comment:
While this solves our problem, it looks wrong in the context of other usage like this that has access to the `Assets` instance directly.